.circleci/config.yml

```diff
@@ -43,27 +49,70 @@ jobs:
           name: "Install Rust Build Toolchain"
           command: |
             curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs > /tmp/rustup-init
-            sh /tmp/rustup-init -y --default-toolchain stable
-            . "$HOME"/.cargo/env
+            for i in $(seq 10); do
+                if sh /tmp/rustup-init -y --default-toolchain stable; then
+                    break
+                else
+                    sleep 1
+                fi
+            done
+            echo '. "$HOME"/.cargo/env' >> $BASH_ENV
 
       - run:
           name: "Get Python Build/Package Dependencies"
           command: |
             # Make sure we have a pip that's aware of Python version constraints
-            pip install --upgrade pip
+            << parameters.python >> -m pip install --upgrade pip
             # Pre-install these two setup_depends using pip so we don't have
             # to rely on whatever resolution logic setuptools would apply.
-            pip install --upgrade milksnake setuptools_scm
-            # And get these so we can build and then upload a wheel
-            pip install wheel twine
+            << parameters.python >> -m pip install --upgrade milksnake setuptools_scm
+            # And get these so we can build, fix, and upload a wheel.
+            << parameters.python >> -m pip install wheel twine auditwheel
 
       - run:
           name: "Build Wheel"
           command: |
-            python setup.py bdist_wheel
+            << parameters.python >> -m pip wheel .
+
+      - run:
+          name: "Audix / Fix Wheel "
+          command: |
+            # Since both macOS and Linux jobs re-use this step, make sure we
+            # only try to use auditwheel on the appropriate platform.  That
+            # is, only on Linux.
+            case "${CIRCLE_JOB}" in
+              *manylinux*)
```

Comment:
I'd really like to suggest that this conditional be moved out-of-shell (there is a `when` step, that could contain a `run` step), but I couldn't figure out if there was a way to have the condition be on the executor, without adding an additional manually specified parameter to this command. Given that, I don't think it is worth the effort to change.

Reply:
I also tried to find a different place to put this logic and came up empty.  Thanks for pointing me at `when` again, though.  On my first look at the docs I got as far as the when clause on `run` that is limited to `always` / `on_success` / `on_fail` and didn't read far enough to see the separate `when` step that has more expressive conditionals.

It seemed like:

```
when:
  condition:
    matches:
      pattern: ".*manylinux.*"
      value: ...
    steps: [ ... ]
```

gets pretty close ... but none of the values containing that string _seem_ to be usable in the match value slot (the docs don't state outright that you can't use environment variables but it seems implied, both by omission and by the statement about when `when` is resolved - "configuration compilation" time).  

However, there are only two "call sites" for `build-wheels` so it might not be that bad to just add the explicit parameter, I suppose.